visivo/command_line.py

```diff
@@ -40,6 +41,9 @@ def load_env(env_file):
 def safe_visivo():
     try:
         visivo(standalone_mode=False)
+    except (ValidationError, LineValidationError) as e:
+        Logger.instance().error(str(e))
+        exit(1)
```

Comment:
This would exit the command with an error on the first validation error found right? It might be better to collect all of the validation errors first and then fail the command so that you don't have the run the command multiple times to find all of your errors.

Reply:
All the parse errors are collected by pydantic before it raises.